types/ngrok.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'ngrok' {
+  namespace Ngrok {
+    interface Tunnel {
+      config: {
```

Comment:
Should this also include the `proto` field we reference on https://github.com/deathandmayhem/jolly-roger/pull/1213/files#diff-5f3a785e0ccb0416158647d90d43c17f2789d516b0d6bdb2ef7f8b0ee0f79e0eR242 ?

Reply:
No, we get that from upstream. The upstream type declarations were just missing `config` (They took a PR but I don't think they've released a new version yet)